src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,17 +1,29 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(cascade = CascadeType.REFRESH,
```

Comment:
обєкт Smile може бути в багатьох Comment, і Comment може мати List<Smile>
```suggestion
    @ManyToMany(cascade = CascadeType.REFRESH,
```